lnrpc/rpc.proto

```diff
@@ -873,8 +884,19 @@ message SendRequest {
     */
     string dest_string = 2 [deprecated = true];
 
-    /// Number of satoshis to send.
-    int64 amt = 3;
+    /**
+    The amount to send expressed in satoshis.
+
+    The fields amt_sat and amt_msat are mutually exclusive.
+    */
+    int64 amt_sat = 3;
```

Comment:
This is a breaking change for users of the API. Should we start adding a "Breaking Changes" section to the release notes?

Reply:
It is a breaking change, but not on the wire level. I will add a comment to the PR description so that it can be picked up when we write the release notes.

Reply:
Ah, right, because the field names aren't sent over the wire (in gRPC). And in REST it's an unnamed parameter in the URL.
My comment about removing all sat fields doesn't make sense then.